webextensions/api/declarativeNetRequest.json

```diff
@@ -0,0 +1,1656 @@
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": "15"
+              },
+              "safari_ios": "mirror"
+            }
+          }
+        },
+        "getMatchedRules": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/declarativeNetRequest/getMatchedRules",
+            "support": {
+              "chrome": {
+                "version_added": "84"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": "15"
+              },
+              "safari_ios": "mirror"
+            }
+          }
+        },
+        "getSessionRules": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/declarativeNetRequest/getSessionRules",
+            "support": {
+              "chrome": {
+                "version_added": "90"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": "15"
+              },
+              "safari_ios": "mirror"
+            }
+          }
+        },
+        "isRegexSupported": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/declarativeNetRequest/isRegexSupported",
+            "support": {
+              "chrome": {
+                "version_added": "87"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": "15"
+              },
+              "safari_ios": "mirror"
+            }
+          }
+        },
+        "onRuleMatchedDebug": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/declarativeNetRequest/onRuleMatchedDebug",
+            "support": {
+              "chrome": {
+                "version_added": "84",
+                "notes": "Available only to unpacked extensions."
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": "15"
```

Comment:
`onRuleMatchedDebug` is not supported by Safari.

Reply:
```suggestion
                "version_added": false
```